src/main/java/net/sf/jabref/external/AccessLinksForEntries.java

```diff
@@ -1,231 +0,0 @@
-            int pos = name.lastIndexOf('.');
-            String extension = ((pos >= 0) && (pos < (name.length() - 1))) ? name.substring(pos + 1)
-                    .trim().toLowerCase() : null;
-
-            // Find the default directory for this field type, if any:
-            String[] dir = metaData.getFileDirectory(extension);
-            // Include the standard "file" directory:
-            String[] fileDir = metaData.getFileDirectory(GUIGlobals.FILE_FIELD);
-            // Include the directory of the bib file:
-            ArrayList<String> al = new ArrayList<String>();
-            for (String aDir : dir) {
-                if (!al.contains(aDir)) {
-                    al.add(aDir);
-                }
-            }
-            for (String aFileDir : fileDir) {
-                if (!al.contains(aFileDir)) {
-                    al.add(aFileDir);
-                }
-            }
-
-            String[] dirs = al.toArray(new String[al.size()]);
-            File tmp = FileUtil.expandFilename(entry.getLink(), dirs);
-            if (tmp != null) {
-                file = tmp;
-            }
-
-            // Check if we have arrived at an existing file:
-            if (file.exists()) {
-                if (fileNames.contains(name)) {
-                    // Oops, a file of that name already exists....
-                } else {
-                    fileNames.add(name);
-                    File destination = new File(toDir, name);
-
-                    // Check if the source and destination locations differ:
-                    if (!destination.equals(file)) {
-                        try {
-                            // Copy the file:
-                            FileUtil.copyFile(file, destination, false);
-                            // Delete the original file if requested:
-                            if (deleteOriginalFiles) {
-                                file.delete();
-                            }
-
-                        } catch (IOException ex) {
-                            ex.printStackTrace();
-                        }
-                    } else {
-                        // Destination and source is the same. Do nothing.
-                    }
-                    // Update progress bar:
-                    i++;
-                    final int j = i;
-
-                    if (prog != null) {
-                        SwingUtilities.invokeLater(new Runnable() {
-
-                            @Override
-                            public void run() {
-                                prog.setValue(j);
-                            }
-                        });
-                    }
-                }
-            } else {
-                // The link could not be resolved to an existing file.
-
-            }
-        }
-
-        if (callback != null) {
-            callback.actionPerformed(null);
-        }
-    }
-
-
-    public static class SaveWithLinkedFiles implements BaseAction {
```

Comment:
As this class is not part of the shipped product because it is never called, we have no knowledge wether this feature works or not. 

I would delete this as well. Reasoning: slime done and becoming productive again is better than adding any features that may contain bugs and we are also not certain whether the user really want such features.